readme.txt

```diff
@@ -1,8 +1,8 @@
 === Genesis Author Pro ===
-Contributors: nick_thegeek, dreamwhisper, laurenmancke, studiopress
+Contributors: nick_thegeek, dreamwhisper, laurenmancke, studiopress, marksabbath, jivedig
 Tags: genesis, authors, books
 Requires at least: 3.9
-Tested up to: 4.7
+Tested up to: 5.0.3
 Stable tag: 1.0
```

Comment:
@marksabbath Should the stable tag read 1.0.1 for release (here and in readme.md)?